backend/src/entities/scene.entity.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  ManyToOne,
+  OneToMany,
+} from 'typeorm';
+import { Movie } from './movie.entity';
+import { Track } from './track.entity';
+
+@Entity()
+export class Scene {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  name: string;
+
+  @ManyToOne(() => Movie, (movie) => movie.scenes)
+  movie: Movie;
+
+  @OneToMany(() => Track, (track) => track.scene)
+  tracks: Track[];
```

Comment:
Correct use of relations to Movie and Track 🔥

Reply:
Something to consider is to add cascade options if you want scenes to be deleted when a movie is deleted. Also to add index on foreign keys for performance 😊